app/helpers/cart_helper.rb

```diff
@@ -1,11 +1,11 @@
 module CartHelper
-  def cart_total(products)
-    #raw_cart_total(products) / 100
+  def cart_total(tickets)
+    raw_cart_total(tickets) / 100
   end
 
-  def raw_cart_total(products)
-    products.map do |product|
-      product.price * product.quantity
+  def raw_cart_total(tickets)
+    tickets.map do |ticket|
+      ticket.price
     end.reduce(:+)
   end
```

Comment:
you can do this in the reduce instead of having to do a map and a reduce 
it would probably be something like tickets.reduce(0){ | sum, ticket | sum+ticket.price }